packages/react-magma-dom/src/components/Dropdown/DropdownMenuItem.tsx

```diff
@@ -176,6 +195,8 @@ export const DropdownMenuItem = React.forwardRef<
       {...other}
       aria-disabled={disabled}
       disabled={disabled}
+      expandableMenuButtonHasIcon={menuGroupContext.expandableMenuButtonHasIcon}
+      isExpandablePanel={menuGroupContext.isExpandablePanel}
```

Comment:
`isExpandablePanel={!!menuGroupContext}` works here, and we can remove `isExpandablePanel` from the context. The reason it works is because, if you are using `DropdownExpandableMenuGroup` at all, the context exists and therefore `isExpandablePanel` is true. We can chat about this on Zoom if you want.

Reply:
Retaining current logic as the context is needed.